internal/lsp/watcher/rlimit_stub.go

```diff
@@ -0,0 +1,12 @@
+//go:build !unix
+
+package watcher
+
+// MaximizeOpenFileLimit is a no-op on non-Unix systems.
+// Returns a high value to indicate no practical limit.
+func MaximizeOpenFileLimit() (int, error) {
```

Comment:
This should not be an exported / public function. Unless we move into `fsext` or something...